web-app-router/components/Radio/index.tsx

```diff
@@ -1,7 +1,63 @@
-export const Radio = () => {
+"use client";
+
+import clsx from "clsx";
+import React, { memo } from "react";
+import { FieldError, UseFormRegisterReturn } from "react-hook-form";
+
+type RadioProps = {
+  register: UseFormRegisterReturn;
+  required?: boolean;
```

Comment:
@andy-t-wang same as [here](https://github.com/worldcoin/developer-portal/pull/472#discussion_r1466495732)

Reply:
make label optional

Reply:
Comment about required is not relevant anymore